File: src/clients/Ruby/take_io_client/entities/message.rb
module TakeIoClient
	module Entities
		class Message < Base
			attribute :id, String
			attribute :schedule, String
			attribute :owner, String
			attribute :sender, String
			attribute :recipients, RecipientsResource
			attribute :contactGroups, ContactGroup
			attribute :body, String
			attribute :subject, String
			attribute :time, DateTime
			attribute :wallet, String
			attribute :urgent, Boolean
			attribute :ackUri, String
			attribute :replyUri, String
			attribute :folder, String
			attribute :status, String
			attribute :category, String
			attribute :type, String
			attribute :validity, Integer
			attribute :created, DateTime
			attribute :updated, DateTime
			attribute :audioBase64, String
			attribute :async, Boolean
			attribute :specificId, String
			attribute :idDomain, String
			attribute :largeAccount, String
			attribute :read, Bool
		end
	end
end

File: src/clients/PHP/TakeIoClient/Entities/Message.php
<?php

namespace TakeIoClient\Entities;

use TakeIoClient\Entities\Base;

class Message extends Base
{
    public $id;
    public $schedule;
    public $owner;
    public $sender;
    public $recipients;
    public $contactGroups;
    public $body;
    public $subject;
    public $time;
    public $wallet;
    public $urgent;
    public $ackUri;
    public $replyUri;
    public $folder;
    public $status;
    public $category;
    public $type;
    public $validity;
    public $created;
    public $updated;
    public $audioBase64;
    public $async;
    public $specificId;
    public $idDomain;
    public $largeAccount;
    public $read;
}
